RayTraceAntiXray/pom.xml

```diff
@@ -96,8 +147,8 @@
               </executableDependency>
               <mainClass>net.fabricmc.tinyremapper.Main</mainClass>
               <arguments>
-                <argument>${project.build.directory}/${project.build.finalName}-mojang-mapped.jar</argument>
-                <argument>${project.build.directory}/${project.build.finalName}.jar</argument>
+                <argument>${project.build.directory}/${project.build.finalName}-spigot-mapped.jar</argument>
+                <argument>${project.build.directory}/${project.build.finalName}-spigot-mapped.jar</argument>
```

Comment:
@stonar96 should it be one with a classifier and the other without it?

Reply:
No this is correct. The artifact created by the Maven Shade Plugin which is classified as spigot-mapped is just a temporary JAR file (which already has the correct manifest entry but still needs to be remapped). Then I use this temporary JAR file as input for tiny-remapper and I specify the same path as output file to overwrite it.

Note that I'm doing a similar thing for Maven's original unclassified artifact already. First the Maven JAR Plugin creates the original unclassified artifact (which needs to be shaded). Then the Maven Shade Plugin picks up this artifact and first creates the spigot-mapped artifact (temporary file, to be remapped) and second the shaded Mojang mapped artifact with the correct manifest entry. In the second step I specify the output file path, so that the original artifact is overwritten. (Note that the order of the two steps matters here. Also note that without specifying the output file path, Maven would keep the original file by renaming it but we don't need it.)